test/java/src/test/java/ghostdriver/AuthBasicTest.java

```diff
@@ -61,4 +81,36 @@ public void simpleBasicAuthShouldWork() {
         assertTrue(driver.getPageSource().contains("authenticated"));
     }
 
+    // we should be able to interact with pages that have content security policies
+    // @Ignore
+    @Test
+    public void canSendKeysAndClickOnPageWithCSP() {
```

Comment:
refactor the CSP test so it does not depend on twitter.  see ariya/phantomjs#14858 for why the test was failing.  phantomjs was creating duplicate requests after the redirect and not setting the proper headers.  hence twitter returned the wrong content type.